documentation/adoc/cluster-controller.adoc

```diff
@@ -404,7 +404,22 @@ env:
 
 
 `STRIMZI_OPERATION_TIMEOUT_MS`:: Optional, default: 60000 ms. The timeout for internal operations, in milliseconds. This value should be
-increased when using Strimzi on clusters where regular Kubernetes operations take longer than usual (because of slow downloading of Docker images, for example). 
+increased when using Strimzi on clusters where regular Kubernetes operations take longer than usual (because of slow downloading of Docker images, for example).
+
+`STRIMZI_DEFAULT_KAFKA_IMAGE`:: Optional, default `strimzi/kafka:latest`.
+The image name to use when deploying Kafka.
```

Comment:
Could we add here _The default image ..._? To make it a bit more clear that this can be overridden in the config map.

Reply:
OK, I've now fixed this, and amended the documentation of the defaults in the respective documentation of the ConfigMaps. I noticed two further things:

1. We specify the topic controller image as the `image` key in the `topic-controller-config`. That's a little inconsistent with the kafka and zk images, which are top level keys in the cluster config map. And it stands out as the only thing in the topic controller config JSON which isn't directly consumed by the TC itself, but it more meta configuration. Limitation of using a CM, I guess. When it's a CRD I suppose it would be some `topicController/spec` with `env` and `image` keys...

2. We don't have any documentation for the Kafka Connect s2i configuration.

Reply:
ad 1) Yeah, I think it would deserve some restructuring. I was thinking about something like this:
```
{
    kafka: {
        image: xxx
        replicas: 5
        healthcheck: {
            xxx
            xxx
        }
        resources: {
            memory_request: xxx
            cpu_request: xxx
            memory_limit: xxxx
        }
        storage {
            type: ephemeral
        }
        metrics: {
            xxx
        }
        config {
            xxx: yyy
        }
    }
    zookeeper: {
        image: xxx
        replicas: 3
        healthcheck: {
            xxx
            xxx
        }
        resources: {
            memory_request: xxx
            cpu_request: xxx
            memory_limit: xxxx
        }
        storage {
            type: persistent
        }
    }
    topic-controller: {
        image: xxx
        healthcheck: {
            xxx
            xxx
        }
        resources: {
            memory_request: xxx
            cpu_request: xxx
            memory_limit: xxxx
        }
        config {
            xxx: yyy
        }
    }
}
```
It didn't come to may mind to model it more after other Kubernetes resources. But it sounds interesting, maybe I should think about it.

Of course ideally we would do this change before next release ... but that doesn't seem realistic :-(.

ad 2) This is covered by #225 - lets do that separately